leo_filters/src/imu_filter_parameters.yaml

```
@@ -42,4 +42,31 @@ imu_filter:
     read_only: true
     validation:
       bounds<>: [1, 600]
-  
+  angular_velocity_threshold:
+    type: double
+    default_value: 0.2
+    description: "Threshold for the angular velocity to determine if the filter is in steady state."
```

Is the filter or the IMU in the steady state?
```suggestion
    description: "Threshold for the angular velocity to determine whether the IMU is in steady state."
```